packages/assets-controllers/src/AssetsContractController.test.ts

```diff
@@ -59,7 +58,7 @@ async function setupControllers() {
   });
   const preferences = new PreferencesController();
   const assetsContract = new AssetsContractController({
-    chainId: ChainId.mainnet,
+    caipChainId: SupportedTokenDetectionNetworks.mainnet,
```

Comment:
The values in [SupportedTokenDetectionNetworks](https://github.com/MetaMask/core/blob/18b64149a46b911d5f26b238e797b163374b2c08/packages/assets-controllers/src/assetsUtil.ts#L143) don't appear to be caip formatted?

Reply:
oh whoops I see they are modified in this PR.